README.md

```diff
@@ -1,5 +1,230 @@
+with [vcpkg](https://github.com/microsoft/vcpkg) or [chocolatey](https://chocolatey.org/).
+
+- Installing `openssl` with `vcpkg`:
+
+```
+./vcpkg.exe install openssl:x64-windows
+./vcpkg.exe integrate install
+# You may want to add this to the system environment variables since you'll need it to compile the crate
+set VCPKGRS_DYNAMIC=1
+```
+
+- Installing `openssl` with `chocolatey`:
+
+```
+choco install openssl
+# You may need to set the OPENSSL_DIR environment variable
+set OPENSSL_DIR="C:\Program Files\OpenSSL-Win64"
+```
+
+#### macOS
+
+You can install `cmake` and `openssl` with `Homebrew`:
+
+```
+brew install cmake openssl@1.1
+```
+
+#### Linux
+
+You can install `cmake`, `clang`, and `openssl` with your distro's package manager or download them from their websites.
+On Debian and Ubuntu, you will also need the `build-essential` and `libudev-dev` packages.
+
+## Getting Started
+
+### Install the IOTA SDK
+
+To start using the IOTA SDK in your Rust project, you can include the following dependencies in your `Cargo.toml` file:
+
+```toml
+[dependencies]
+iota-sdk = { branch = "develop" }
+```
+
+### Usage
+
+#### Wallet
+
+To use the Wallet module, you need to create a `Wallet`:
+
+```rust
+use iota_sdk::{
+    client::{
+        constants::SHIMMER_COIN_TYPE,
+        secret::{stronghold::StrongholdSecretManager, SecretManager},
+    },
+    wallet::{ClientOptions, Result, Wallet},
+};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Setup Stronghold secret manager.
+    let secret_manager = StrongholdSecretManager::builder()
+        .password("vault.stronghold") // A password to encrypt the stored data.WARNING: Never hardcode passwords in production code.
+        .build(PathBuf::from("vault.stronghold"))?; // The path to store the account snapshot.
+
+    let client_options = ClientOptions::new().with_node("https://api.testnet.shimmer.network")?;// The node to connect to.
+
+    // Set up and store the wallet.
+    let wallet = Wallet::builder()
+        .with_secret_manager(SecretManager::Stronghold(secret_manager))
+        .with_client_options(client_options)
+        .with_coin_type(SHIMMER_COIN_TYPE)
+        .finish()
+        .await?;
+
+    // Generate a mnemonic and store it in the Stronghold vault.
+    // INFO: It is best practice to back up the mnemonic somewhere secure.
+    let mnemonic = wallet.generate_mnemonic()?;
+    wallet.store_mnemonic(mnemonic).await?;
+
+    // Create an account.
+    let account = wallet
+        .create_account()
+        .with_alias("Alice".to_string()) // A name to associate with the created account.
+        .finish()
+        .await?;
+
+
+    Ok(())
+}
+```
+
+#### Client
+
+To use the Client module, you simply need to create a `Client`.
+
+
+```rust
+use iota_sdk::client::{
+    Client,
+};
+
+let client = Client::builder()
+    .with_node('https://api.testnet.shimmer.network')? // Insert your node URL here
+    .finish()
+    .await?;
```

Comment:
Should this maybe also print the nodeinfo? Just building the client is not that interesting